### ISSUE TITLE
Instancio.create(<some, but not all classes>) returns null, ~no error~.

### ISSUE THREAD

**Issue by drewdiddy611** [instancio/instancio#992]
**Describe the bug**
Title describes it all.  
- `Instancio.create(<some class>); // returns instance of that class`
- `Instancio.create(<some other class>); // returns null.  No error, no nothing, just returns null.`

**Expected result**
Should return an instance of that object.  

**Actual result**
- the `some class` type is basically identical in form (default constructor, just a simple POJO with fields, setters and getters, etc) to `some other class`, but `some other class` returns null.  No error in the console.

**Additional information**
Looked at the docs, couldn't find anything.  Googled it, without much success, because I only get results about "Java Instances," likely due to the library itself being named similar to a Java fundamental concept.

Both classes inherit (eventually) from the same root class.

**Link to a minimal sample project reproducing the issue or runnable sample code**
Can't link or provide code sample, proprietary code.  

Can we at least perform a null check on the return value and provide a reason why it could ever be null?  Maybe some documentation on scenarios where it could return null.  I couldn't find any.

Thanks

- Java version: 17
- Instancio version: 4.5.0

**Comment by drewdiddy611** [comment]
Screenshot of invocation returning null:
<img width="1131" alt="image" src="https://github.com/instancio/instancio/assets/3035686/3ef9c858-a7a2-433a-85a0-ca999325f6b9">

**Comment by armandino** [comment]
@drewdiddy611 thanks for reporting this issue. It's hard to say what causes it without a sample. Any chance you could rename the POJO classes and remove irrelevant fields/methods?

Does running the code in `verbose()` mode provide any additional information?

```java
SomePojo pojo = Instancio.of(SomeoPojo.class)
    .verbose()
    .create();
```

**Comment by drewdiddy611** [comment]
> @drewdiddy611 thanks for reporting this issue. It's hard to say what causes it without a sample. Any chance you could rename the POJO classes and remove irrelevant fields/methods?
> 
> Does running the code in `verbose()` mode provide any additional information?
> 
> ```java
> SomePojo pojo = Instancio.of(SomeoPojo.class)
>     .verbose()
>     .create();
> ```

@armandino
Thanks for the quick reply!  And for the suggestion about verbose(), I missed that one.  Here is the error:

```
Exception occurred while generating the root object of type EquipmentMake.
Printing stacktrace because verbose() mode is enabled.
To propagate internal errors, set Keys.FAIL_ON_ERROR setting to true.
```
`java.lang.NullPointerException: Cannot invoke "String.hashCode()" because "this.namespaceURI" is null`

Here is the root class (which is what I *think* it's refererring to?):
```java
@XmlAccessorType(XmlAccessType.FIELD)
        @XmlType(
                name = "RootClass",
                propOrder = {...}
        )
        @XmlSeeAlso({...})
        public class RootClass {
            @XmlElement(
                    name = "..."
            )
            protected String a;
            @XmlElement(
                    name = "..."
            )
            protected boolean b;
            @XmlElement(
                    name = "..."
            )
            protected List<Property> c;
            @XmlElement(
                    name = "...",
                    required = true,
                    nillable = true
            )
            protected String d;
            @XmlElement(
                    name = "...",
                    required = true,
                    nillable = true
            )
            @XmlSchemaType(
                    name = "dateTime"
            )
            protected XMLGregorianCalendar e;
            @XmlElement(
                    name = "...",
                    required = true,
                    nillable = true
            )
            protected String f;
            @XmlElement(
                    name = "...",
                    required = true,
                    nillable = true
            )
            protected String g;
            @XmlElement(
                    name = "...",
                    required = true,
                    nillable = true
            )
            @XmlSchemaType(
                    name = "dateTime"
            )
            protected XMLGregorianCalendar h;
            @XmlElement(
                    name = "...",
                    required = true,
                    nillable = true
            )
            protected String i;
            @XmlElement(
                    name = "...",
                    required = true,
                    nillable = true
            )
            protected String j;
            @XmlElement(
                    name = "...",
                    required = true,
                    nillable = true
            )
            protected String k;
            @XmlElement(
                    name = "..."
            )
            protected boolean l;

            public RootClass() {
            }

            public String getA() {
                return this.a;
            }

            public void setA(String value) {
                this.a = value;
            }

            public boolean isB() {
                return this.b;
            }

            public void setB(boolean value) {
                this.b = value;
            }

            public List<Property> getC() {
                if (this.c == null) {
                    this.c = new ArrayList();
                }

                return this.c;
            }

            public String getD() {
                return this.d;
            }

            public void setD(String value) {
                this.d = value;
            }

            public XMLGregorianCalendar getE() {
                return this.e;
            }

            public void setE(XMLGregorianCalendar value) {
                this.e = value;
            }

            public String getF() {
                return this.f;
            }

            public void setF(String value) {
                this.f = value;
            }

            public String getG() {
                return this.g;
            }

            public void setG(String value) {
                this.g = value;
            }

            public XMLGregorianCalendar getH() {
                return this.h;
            }

            public void setH(XMLGregorianCalendar value) {
                this.h = value;
            }

            public String getI() {
                return this.i;
            }

            public void setI(String value) {
                this.i = value;
            }

            public String getJ() {
                return this.j;
            }

            public void setJ(String value) {
                this.j = value;
            }

            public String getK() {
                return this.k;
            }

            public void setK(String value) {
                this.k = value;
            }

            public boolean isL() {
                return this.l;
            }

            public void setL(boolean value) {
                this.l = value;
            }
        }
```

**Comment by drewdiddy611** [comment]
@armandino Update:
I think it's this `Map<QName, String>` field on the class:
```java
        @XmlAccessorType(XmlAccessType.FIELD)
        @XmlType(
                name = "EquipmentMake"
        )
        public class EquipmentMake extends UniqueReferenceEntity {
            @XmlAnyAttribute
            private Map<QName, String> A = new HashMap();

            public EquipmentMake() {
            }

            public Map<QName, String> getA() {
                return this.A;
            }
        }
```

**Comment by drewdiddy611** [comment]
Fixed by setting the map to an empty map:

`instancioOfClassApi.set(fields().ofType(Map.class).named("A"), emptyMap());`

**Comment by armandino** [comment]
@drewdiddy611  thank you for the follow up. I suspect `QName` is causing the problem because it's from a `javax` package and does not have a built-in generator defined. I will give it a try to confirm. If that's the case, I'll create a bug fix so that you don't need to `ignore()` the map.

**Comment by armandino** [comment]
@drewdiddy611 I just released a new version `4.5.1` which should fix the root cause. This should remove the need for the workaround of setting an empty map. Please let me know if you run into any other issues.
